ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-multicast.j2

```diff
@@ -1,17 +1,40 @@
-{% if router_multicast is defined and router_multicast is not none %}
+{% if router_multicast is arista.avd.defined %}
 
 ## Router Multicast
 
 ### IP Router Multicast Summary
 
-{%   if router_multicast.ipv4.routing is defined and router_multicast.ipv4.routing == true %}
-Routing for IPv4 multicast is enabled.
+{%     if router_multicast.ipv4.routing is arista.avd.defined(true) %}
+- Routing for IPv4 multicast is enabled.
+{%     endif %}
+{# Multipathing deterministically picking would take precedence based on order of commands #}
+{%     if router_multicast.ipv4.multipath_deterministic_pick is arista.avd.defined %}
+- Multipathing deterministically picking {{ router_multicast.ipv4.multipath_deterministic_pick }}.
+{%     elif router_multicast.ipv4.multipath is arista.avd.defined('none') %}
+- Multipathing disabled.
+{%     elif router_multicast.ipv4.multipath is arista.avd.defined('deterministic') %}
+- Multipathing via ECMP.
+{%     endif %}
+{%     if router_multicast.ipv4.software_forwarding is arista.avd.defined('kernel') %}
+- Software forwarding by the Linux kernel
+{%     elif router_multicast.ipv4.software_forwarding is arista.avd.defined('sfe') %}
+- Software forwarding by the Software Forwarding Engine (SFE)
+{%     endif %}
+
+### IP Router Multicast VRFs
+
+{%     if router_multicast.vrfs is arista.avd.defined %}
+| VRF Name | Multicast Routing |
+| -------- | ----------------- |
+{%         for vrf in router_multicast.vrfs| arista.avd.natural_sort('name') %}
+| {{ vrf.name }} |{% if vrf.ipv4.routing is arista.avd.defined(true) %} enabled {% else %} disabled {% endif %}|
```

Comment:
I am not really sure but I guess we are avoiding inline if else to allow future developments.